### PR TITLE
feat: Dynamic og image

### PR DESCRIPTION
**Pull request by andreifilip123**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

---

## Screenshots

image

There's place for improvement on the UI but the functionality works

💯

### CODE REVIEW

File: upgrade/src/pages/api/og.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { ImageResponse } from "@vercel/og";
```
Comment (juliusmarminge) [inline upgrade/src/pages/api/og.tsx:1]: ImageResponse is bundled in next/server

No need to install @vercel/og explicitely

File: upgrade/src/pages/api/og.tsx
Comment (juliusmarminge) [inline upgrade/src/pages/api/og.tsx:1]: Route handler?

**Comment by andreifilip123** [comment]
Fixed the PR comments + added back og image for the / page (without diff)

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -281,7 +281,7 @@ importers:
         version: 2.8.8
       prettier-plugin-tailwindcss:
         specifier: ^0.2.8
-        version: 0.2.8(@ianvs/prettier-plugin-sort-imports@3.7.2)(prettier-plugin-astro@0.8.1)(prettier@2.8.8)
+        version: 0.2.8(prettier@2.8.8)
```
Comment (andreifilip123) [inline pnpm-lock.yaml:284]: Hm, not sure why the pnpm-lock is updated. Should I revert this ?

File: upgrade/src/app/api/og/route.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                  </p>
+                  <div tw="flex space-x-2">
+                    <span tw="text-green-500">+{additions}</span>
+                    <span tw="text-red-500">-{removals}</span>
+                    <div tw="flex h-4 w-[100px] gap-1 align-middle">
+                      <div tw="flex-1 bg-red-500"></div>
+                      <div tw="flex-1 bg-green-500"></div>
+                      <div tw="flex-1 bg-red-500"></div>
+                      <div tw="flex-1 bg-green-500"></div>
+                      <div tw="flex-1 bg-red-500"></div>
```
Comment (andreifilip123) [inline upgrade/src/app/api/og/route.tsx:53]: Need to compute this based on additions and removals

**Pull request merged**